test/Microsoft.ML.Sweeper.Tests/TestSweeper.cs

```diff
@@ -0,0 +1,756 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Sweeper;
+using Microsoft.ML.Runtime.Sweeper.Algorithms;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Sweeper.RunTests
+{
+    using Float = System.Single;
```

Comment:
>Float [](start = 10, length = 5)

Feel free to remove all Float and replace them with float.
This is legacy patter from times we support Single and Double builds, and can be switch through #ifdef conditions

Reply:
Yes, please use the C# keywords in the code - like `float`, `double`, etc.

---
In reply to: [213881195](https://github.com/dotnet/machinelearning/pull/768#discussion_r213881195) [](ancestors = 213881195)